### PR TITLE
[ICDS Dashboard] Hide the tabular report current month filter for first 3 days

### PR DESCRIPTION
**Pull request by challabeehyv**
##### SUMMARY
current month has been hidden from month filter if the present day is first three days of month

##### FEATURE FLAG
No

### CODE REVIEW

**Review by calellowitz — CHANGES_REQUESTED** [review]
This looks fine, but can you add a test that makes sure the current month will appear after the 3rd

**Comment by challabeehyv** [comment]
> This looks fine, but can you add a test that makes sure the current month will appear after the 3rd

added

File: custom/icds_reports/static/js/directives/download/download.directive.js [hunk 1]
```diff
@@ -414,8 +426,8 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
         }
     };
 
-    vm.setMonthToPreviousIfBeforeThe15th = function (date) {
-        var offset = date.getDate() < 15 ? 1 : 0;
+    vm.setMonthToPreviousIfAfterThe15thAndTwoMonthsIfBefore15th = function (date) {
```
Comment (Rohit25negi) [inline custom/icds_reports/static/js/directives/download/download.directive.js]: Hey, What is this doing actually?. I think there has to be no change in the how performance report works. It never shows the filter for current month.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Generally looks good. Left one question

File: custom/icds_reports/static/js/directives/download/download.directive.js (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/static/js/directives/download/download.directive.js]: we have to show before last month report if present day is first 15 days of the month or else we need to show last month report

**Review by calellowitz — APPROVED** [review]

**Pull request merged**